### PR TITLE
Revoking Hackbot from a team no longer raises errors

### PR DESCRIPTION
**Pull request by MaxWofford**
This fixes a problem where Hackbot doesn't stop crying after the other kids
kick it out of their playground.

### CODE REVIEW

**Comment by MaxWofford** [comment]
This fixes https://sentry.io/hack-club/api/issues/302136093/

File: api/app/jobs/update_hackbot_slack_username_job.rb [hunk 1]
```diff
@@ -4,7 +4,11 @@ class UpdateHackbotSlackUsernameJob < ApplicationJob
   def perform(*)
     Hackbot::Team.find_each do |team|
       info = ::SlackClient::Users.info(team.bot_user_id, team.bot_access_token)
-      team.update(bot_username: info[:user][:name])
+
+      # https://api.slack.com/methods/users.info
+      unless info[:error] == 'account_inactive'
+        team.update(bot_username: info[:user][:name])
```
Comment (paked) [inline api/app/jobs/update_hackbot_slack_username_job.rb]: What was the justification for failing silently and not logging here?

Comment (MaxWofford) [inline api/app/jobs/update_hackbot_slack_username_job.rb]: Just added a `logger.info`

**Pull request merged**